test/youtube_test.py

```diff
@@ -234,6 +235,14 @@
     ),
 ]
 
+SHORTS = [
```

Comment:
Why not include this in `PARSE_TESTS` and drop the `test_parse_short_url` method?